src/Common/src/TypeSystem/Common/TypeHashingAlgorithms.cs

```diff
@@ -208,6 +208,18 @@ public static int ComputeGenericInstanceHashCode<ARG>(int genericDefinitionHashC
             return (hashcode + _rotl(hashcode, 15));
         }
 
+        public static int ComputeMethodSignatureHashCode<ARG>(int returnTypeHashCode, ARG[] parameters)
+        {
+            // TODO: Do we need calling conventions in the mix too?
```

Comment:
Changing these hash functions after the fact may be awkward. Please add them now if its convenient.